### ISSUE TITLE
Long scan times

### ISSUE THREAD

**Issue by narcoma** [shirsig/aux-addon#94]
Hey,

just interested if the full scans usually take the time they need for me, like, far over half an hour for a full scan. I've seen people post about how the do such a full scan everday, which makes me wonder if they know something I don't. Somewhere else I heard that the Vanilla API is blocking you from fast scanning, which would explain it.

I tried ignoring auction owners, it just doesn't seem to make a difference at all (playing on Elysium right now). 

Basically, I'm looking for a hint how long a full scan should take with AUX and how I can speed it up if it is as slow as it is now.

**Comment by shirsig** [comment]
You can only get a page every ~4 seconds from the server. Nothing you can do to improve this apart from the ignore owner mode as the bulk of the time is spent waiting for this timer with the actual processing only taking a fraction of a second.
